README.md

```diff
@@ -1 +1,21 @@
 # orodha-notification-service
+
+This service will handle the creation as well as storage of notifications for the Orodha list app.
+
+## Usage & Requirements
+
+In order to use this repository there are some conditions that have to be met. You must have a `.env` file containing:
+
+- DOCKER_USERNAME
+- DOCKER_PASSWORD
+- TAG
+- DBROOTUSER
+- DBROOTPASSWORD
+- DBUSER
+- DBPASSWORD
+- DBNAME
+  The `docker-compose.main.yaml` file uses these variables in order to build the image using the Dockerfile as well as set up the database.
+
+The Dockerfile has an argument called `BUILD_FILE` that is by default set to `requirements.txt`. For now, this can only be changed by setting an environment variable named BUILD_FILE to the requirement file that you would like to use.
```

Comment:
@Edison-Stuart Might want to change `BUILD_FILE` to `REQUIREMENTS_FILE` for clarity.